libr/cons/cons.c

```diff
@@ -1322,3 +1322,50 @@ R_API void r_cons_breakword(const char *s) {
 		I.break_word_len = 0;
 	}
 }
+
+/* Prints a coloured help message.
+ * help should be an array of the following form:
+ * {"command", "args", "description",
+ * "command2", "args2", "description"}; */
+R_API void r_cons_cmd_help(const char *help[], bool use_color) {
+	int i, max_length, padding_length;
+	RCons *cons = r_cons_singleton ();
+	char const *args_color_start;
+	char const *help_color_start;
+	char const *reset_colors;
+	char padding[256];
+
+	args_color_start = use_color ? cons->pal.args: "";
+	help_color_start = use_color ? cons->pal.help: "";
+	reset_colors     = use_color ? cons->pal.reset: "";
+
+	max_length = 0;
+	for (i = 0; help[i]; i += 3) {
```

Comment:
may crash if not checking null on +1 and +2

Reply:
I know it'll crash, but with the current code you can catch when help messages were not written correctly.